### DISCUSSION TITLE
What do you love about Quire?

### DISCUSSION THREAD

**Discussion by Erin-Cecele** [thegetty/quire#222] [Feedback]
Have you had a great experience with Quire this past year? We’d love to hear about it. As we prepare a marketing push for Quire's official v1 launch in 2023, we are looking for user testimonials to share with the public.

- Is there Quire feature or function you particularly liked?
- Did Quire solve a specific problem for you or your institution? If so, how?
- What do you like best about being part of the community?
- How did Quire help you achieve success?
- What excites you most about Quire?
- How does Quire differ from other publishing tools you’ve tried?

Thanks for sharing your thoughts with us!

**Comment by shineslike** [comment]
In early 2019, several colleagues and I used Quire to make *Humanizing the Digital: Unproceedings from the 2018 MCN Confererence*, a rapidly-produced publication that responded to the MCN annual conference. *Humanizing the Digital* contained 17 reflections, case studies, conversations, essays, and an experimental in-book zine, and marked a specific moment in time. 

Inspired by that project, my *Museums and Digital Technology* class at The George Washington University has started to produce an annual publication using the Quire platform. Last year, *The State of Museum Digital Practice 2021: A collection of graduate essays and responses* was written, edited and published by students over a ten-week period during the Fall 2021 semester. The project aims to give students a deep engagement with a topic of personal or professional interest through a research project; experience in a collaborative creative environment through the peer review process; and practical skills in Markdown language. 

Quire has been the perfect vehicle for this project. The platform is easy to use (particularly once you've spent some time with it), the guidelines and documentation are hugely helpful, and working with Greg and Erin from the Getty is a delight. My students gained new skills in Markdown, which was empowering, and I was able to put together the entire publication myself in only a few days of concentrated work (despite only having limited tech-savvy). The results are beautiful, and I am so glad that my students now have a professional-looking publication to share with future employers, peers and loved ones.

I am already planning to use Quire for future teaching, and for a few other projects.
